### PR TITLE
test(demo-homepage): updated demo homepage, removed search bar

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #2863  

## Summary

Restore demo-homepage updates. cc @julio8a

### CODE REVIEW

**Comment by jcfranco** [comment]
#2984 fixes the test failures that came up in #2883. 

**Note**: #2984 should get merged before this PR.

**Comment by jcfranco** [comment]
#2984 is installed!

**Review by driskull — CHANGES_REQUESTED** [review]
@jcfranco can you remove the observer stuff from here?

File: src/demos/_assets/demos.css [hunk 1]
```diff
@@ -38,3 +38,55 @@ main {
...
+  }
+
+  .calcite-comp {
+    color: var(--calcite-ui-text-3);
+    font-family: var(--calcite-sans-family);
+    font-size: var(--calcite-font-size-0);
+    font-weight: var(--calcite-font-weight-medium);
+  }
+
+  a {
```
Comment (driskull) [inline src/demos/_assets/demos.css:89]: Should this selector be scoped? it may mess with the style of some slotted links inside components.

**Review by driskull — APPROVED** [review]
👍

**Review by julio8a — APPROVED** [review]

File: src/demos/_assets/demos.css (hunk 1 above)
Comment (jcfranco) [reply src/demos/_assets/demos.css:89]: @julio8a Can you follow-up on this one? This PR should no longer blow up the build. 🚫 💥

Comment (julio8a) [reply src/demos/_assets/demos.css:89]: I'd rather @caripizza take a look so things don't 💥 again. 😁

@caripizza, can you take a look please?

Comment (caripizza) [reply src/demos/_assets/demos.css:89]: Agree with Matt, do you want me to fix that? I can just move the style to the head of the index html page if that works.

Comment (jcfranco) [reply src/demos/_assets/demos.css:89]: @caripizza Yes, please. I'll hand off this PR to your capable hands. 👐🏼

**Comment by caripizza** [comment]
The Screener failure showing is tied to Popover's Simple story. It seems unrelated to changes in this pr so I'm just going to merge:

image

**Pull request merged**